tarantula.lic

```diff
@@ -0,0 +1,52 @@
+
+custom_require.call(%w[common drinfomon])
+
+class Spider
+  include DRC
+
+  settings = get_settings
+  spider_noun += settings.spider_noun
+  crafting_skills += settings.crafting_skills
+  hunting_skills += settings.hunting_skills
+  outdoor_skills += settings.outdoor_skills
+
+  loop do
+    if Script.running?('workorders') || Script.running?('craft')
+      crafting_skills.each { |e| DRC.bput("turn my bio tarantula to #{e}", /You fiddle/, /With a/)
+      case DRC.bput("rub my #{spider_noun}", /The tarantula comes alive/, /It is mere/, /you currently aren\'t/, "It needs approximately \d+ (?:roisan|roisaen)", /You try, but it does nothing./)
+      when /It needs approximately (\d+) (?:roisan|roisaen)/
+        ptime = Regexp.last_match(1).to_i
```

Comment:
Could you make this (and the next) variable names clearer?

Reply:
Sure - ptime = pause time.